docs/_plugins/include_example.rb

```diff
@@ -0,0 +1,70 @@
+require 'octopress-code-highlighter'
+require 'liquid'
+
+module Octopress
+  module IncludeExample
+    class Tag < Liquid::Tag
+      
+      # Pattern for {% include_example /path/to/a/file.lan %}
+      FileOnly = /^(\S+)$/
+      # Pattern for {% include_example /path/to/a/file.lan title %}
+      FileTitle = /(\S+)\s+(\S.*?)$/i
+
+      def initialize(tag_name, markup, tokens)
+        @markup = markup
+        super
+      end
+
+      def render(context)
+        site = context.registers[:site]
+        config_dir = (site.config['code_dir'] || '../examples/src/main').sub(/^\//,'')
+        @code_dir = File.join(site.source, config_dir)
+
+        begin
+          options = get_options
+          code = File.open(@file).read.encode("UTF-8")
+          code = select_lines(code, options)
+
+          CodeHighlighter.highlight(code, options)
```

Comment:
So there is no need to call outside function, just translate it into markdown highlight. Then we can avoid to use Octopress in the beginning. I get it.